include/queue.php

```diff
@@ -29,6 +29,25 @@ function queue_run(&$argv, &$argc) {
 		// Handling the pubsubhubbub requests
 		proc_run(array('priority' => PRIORITY_HIGH, 'dont_fork' => true), 'include/pubsubpublish.php');
 
+		$interval = ((get_config('system','delivery_interval') === false) ? 2 : intval(get_config('system','delivery_interval')));
+
+		// If we are using the worker we don't need a delivery interval
```

Comment:
I guess - like several parts above - this is some merge problem that reverted some stuff.